cypress/support/components/actionspage.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+/** Navigate to Actions Page */
+function NavigateTo() {
+  cy.get('a[href$="/actions"]')
+    .click();
+}
+/** Click on create a new action button */
+function CreateNew() {
+  cy
+    .get('[data-testid="actions-button-create"]')
+    .click();
+}
+/** Selects Action Type = Text */
+function SelectTypeText() {
```

Comment:
These functions SelectTypeText, SetPhrase and Save are not part of the Actions page. They are part of the Action modal